templates/admin/payments-details.tpl.php

```diff
@@ -85,6 +85,12 @@
                                     <label><?php esc_html_e( 'Gateway', 'business-directory-plugin' ); ?></label>
                                     <?php /* translators: Gateway: (Not yet set) */ ?>
                                     <?php echo esc_html( $payment->gateway ? $payment->gateway : __( '(Not yet set)', 'business-directory-plugin' ) ); ?>
+                                    <?php
+                                        printf(
+                                            '<span class="wpbdp-payment-test-mode-label">%s</span>',
```

Comment:
This would be more consistent with the existing codde if we use inline php inside the html.
```
<span class="wpbdp-payment-test-mode-label">
<?php stuff here ?>
</span>
```

Reply:
@stephywells yea that would be.. I just hate html within php, but for consistency I will change it